master/internal/rm/kubernetesrm/spec.go

```diff
@@ -35,8 +35,16 @@ import (
 const (
 	coscheduler = "coscheduler"
 
-	gcTask  = "gc"
-	cmdTask = "cmd"
+	gcTask              = "gc"
+	cmdTask             = "cmd"
+	userLabel           = "determined.ai/user"
+	workspaceLabel      = "determined.ai/workspace"
+	resourcePoolLabel   = "determined.ai/resource_pool"
+	taskTypeLabel       = "determined.ai/task_type"
+	taskIDLabel         = "determined.ai/task_id"
+	experimentIDLabel   = "determined.ai/experiment_id"
+	trialRequestIDLabel = "determined.ai/trial_request_id"
```

Comment:
I don't think we should set requestID. 

RequestID can be shared between different trials in different experiments so I think this field will only cause confusion.